sortbootorder.c

```diff
@@ -164,7 +164,12 @@ int main(void) {
 				break;
 			default:
 				if (key >= 'a' && key <= 'm' ) {
-					move_boot_list( bootlist, get_line_number(max_lines,key), max_lines );
+					u8 line_start = 0;
```

Comment:
@macpijan IIRC most of sortbootorder code follow C89 standard, which requires variable declaration on top of the function. We should keep code style, if there is no important reason for not doing so. Please move this local declarations to top of the function.